cypress/integration/Confirm.feature

```diff
@@ -0,0 +1,127 @@
+Feature: Confirm component
+  I want to change Confirm component properties
+
+  Background: Open Confirm component page
+    Given I open "Confirm" component page
+
+  @positive
+  Scenario Outline: Change data in inner context in Confirm dialog
+    When I set children to "<children>"
+      And I set cancelButton to "<cancelButton>"
+      And I set confirmButton to "<confirmButton>"
+      And I click on a openButton
+    Then dialog inner context children on preview is "<children><cancelButton><confirmButton>"
+    Examples:
```

Comment:
I'm not sure how readable this is. Will this cycle through all the below options?

Reply:
Yes, this TC will check all change combinations for index context in Confirm preview

Reply:
We should prioritise testing characters from languages that our app caters to: i.e. German and Spanish chars etc.

Reply:
replaced the Chinese / Cyrylic characters onto Spanish/German